<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red;
border-color: rgb(0, 0, 198);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="backhomepage">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	
	border-color: rgb(255, 255, 0);
	border-radius:20
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: rgb(0, 255, 255);
border-color: rgb(0, 0, 0);
font: 26pt &quot;MS Shell Dlg 2&quot;;
}
QLabel:hover{
	color: rgb(255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>Artists</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1371</width>
      <height>831</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red;
border-color: rgb(0, 0, 198);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1367</width>
       <height>827</height>
      </rect>
     </property>
     <widget class="QPushButton" name="click1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	border-radius:20;
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>Amit Trivedi</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>47</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="click3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	border-color: rgb(255, 255, 0);
	border-radius:20;
}</string>
      </property>
      <property name="text">
       <string>Arijit Singh</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>47</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="click2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>590</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	border-radius:20;
border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>AR Rahman</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>47</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="click4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>260</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	border-radius:20;
border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>Ayushmann Khuranna</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>47</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="click5">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>260</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	border-radius:20;
border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>Divine</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>47</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="click6">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>260</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	border-radius:20;
border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>KSHMR</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>47</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="click7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>590</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	border-radius:20;
border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>Mohit Chauhan</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>47</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="click8">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>590</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background: transparent;
color: rgb(0, 255, 255);
	border-color: rgb(0, 255, 255);
font:15pt &quot;Arial Black&quot;;
border-radius:20

}

QPushButton:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 255, 255);
	border-radius:20;
border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>Vishal Dadlani</string>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>47</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-55</y>
        <width>1371</width>
        <height>831</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/photo-1535007726788-fed8106a64cd.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>221</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border-color: rgb(0, 255, 255);
}
QLabel:hover{
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/jMuAw4IQ_400x400.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>211</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border-color: rgb(0, 255, 255);
}
QLabel:hover{
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/69281268.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>241</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border-color: rgb(0, 255, 255);
}
QLabel:hover{
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/ayush.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>211</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border-color: rgb(0, 255, 255);
}
QLabel:hover{
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/images (1).jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>221</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border-color: rgb(0, 255, 255);
}
QLabel:hover{
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/pjimage-1582246037.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>10</y>
        <width>201</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border-color: rgb(0, 255, 255);
}
QLabel:hover{
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/KSHMR-2019-bbrc-billboard-1548-compressed.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>211</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border-color: rgb(0, 255, 255);
}
QLabel:hover{
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/52191496-best-songs-by-mohit-chauhan-640x480-1.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>340</y>
        <width>231</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border-color: rgb(0, 255, 255);
}
QLabel:hover{
	border-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Images/images (2).jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>click1</zorder>
     <zorder>click3</zorder>
     <zorder>click2</zorder>
     <zorder>click4</zorder>
     <zorder>click5</zorder>
     <zorder>click6</zorder>
     <zorder>click7</zorder>
     <zorder>click8</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Images/mp.qrc"/>
 </resources>
 <connections/>
</ui>
